### PR TITLE
doc and config updates for the v0.12.0 release

### PR DESCRIPTION
**Pull request by rolinh**
See commits for details.

### CODE REVIEW

**Review by gandro — APPROVED** [review]

File: .github/renovate.json5 [hunk 1]
```diff
@@ -136,9 +136,9 @@
         "go",
         "golang"
       ],
-      "allowedVersions": "<=1.19",
+      "allowedVersions": "<=1.20",
```
Comment (kaworu) [inline .github/renovate.json5:139]: Would the following work?

```suggestion
      "matchUpdateTypes": ["patch", "digest", "pin", "pinDigest"],
```

If yes then we wouldn't have to track the Go version every time we branch to a new minor version.

File: .github/renovate.json5 [hunk 2]
```diff
@@ -154,9 +154,9 @@
       "matchPackageNames": [
         "docker.io/library/alpine"
       ],
-      "allowedVersions": "<=3.17",
+      "allowedVersions": "<=3.18",
```
Comment (kaworu) [inline .github/renovate.json5:157]: ditto

```suggestion
      "matchUpdateTypes": ["patch", "digest", "pin", "pinDigest"],
```

File: README.md
Comment (kaworu) [inline README.md:1]: Do we start removing old versions at some point?

**Review by kaworu — APPROVED** [review]
Couple of nitpicking questions

**Comment by rolinh** [comment]
@kaworu I merged the PR since the questions/suggestions are not related to the release itself. Would you mind opening a PR with the proposed changes to discuss?

**Comment by kaworu** [comment]
@rolinh opened https://github.com/cilium/hubble/pull/1122

**Pull request merged**